### DISCUSSION TITLE
Github Account suspended without a suspension letter

### DISCUSSION THREAD

**Discussion by Shubham11440** [community/community#129344] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello,
I'm reaching out for help regarding my GitHub account, which has been unexpectedly suspended. I've encountered this issue recently and am quite concerned, as I haven't received any suspension letter or explanation for this action.
Key points:

My account is currently suspended
I did not receive any notification or explanation for the suspension
To my knowledge, I haven't violated any GitHub terms or community guidelines

I'm hoping someone from the community or GitHub staff can provide guidance on:

Why my account might have been suspended without notice
The proper channels to inquire about and appeal this suspension
Any steps I can take to have my account reviewed and potentially reinstated

If any GitHub staff members are monitoring this forum, I would greatly appreciate your input or assistance in resolving this matter. I'm more than willing to provide any necessary information to help clarify the situation.
Thank you in advance for any help or advice you can offer.

Ticket no. - #2849590 Username - Shubham6621

**Comment by zecoryx** [comment]
Response #3

**Comment by ebndev** [comment, marked as answer]
Hi @Shubham11440,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this, and any duplicate discussions. We appreciate your understanding!
